backend/src/main/kotlin/com/infowings/catalog/data/objekt/ObjectDaoService.kt

```diff
@@ -107,30 +106,36 @@ class ObjectDaoService(private val db: OrientDatabase) {
                 DetailedRootValueViewResponse(
                     rootValue.id,
                     rootValue.toObjectPropertyValue().calculateObjectValueData().toDTO(),
-                    rootValue.measure?.toMeasure()?.symbol,
+                    rootValue.explicitMeasureSymbol(),
                     rootValue.description,
```

Comment:
I just realized that `getOrCalculateMeasureSymbol()` could be more appropriate for now. This is because the function either returns measure symbol from measure that is directly attached to value or calculates the measure based on the corresponding aspect